src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -12,21 +12,56 @@
 </template>
 
 <script lang="ts">
-import { mapGetters } from 'vuex'
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Vue, Watch } from 'vue-property-decorator'
 import Block from '@/components/interaction-designer/Block.vue'
+import { namespace } from 'vuex-class'
+import {IBlock, IFlow} from '@floip/flow-runner'
+import { find, isEqual, cloneDeep } from 'lodash'
+import { IValidationStatus } from '@/store/validation'
```

Comment:
@safydy-r  since we changed ValidationStatus to ValidStatus, can we change the interface name as well here?